### PR TITLE
feat: navbar converted to sticky

### LINKED ISSUE #29: Navigation: Expand Horizontally, Fix BG
The top navigation should expand all the way horizontally and not show background like this:
Screenshot 2023-01-17 at 11 28 56

Also, the navigation should continue to show a small shadow when scrolling happened underneath:
Screenshot 2023-01-17 at 11 29 24

### PR DESCRIPTION
**Pull request by ekremney**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #29 

Test URLs:
- Before: https://main--elco-aerotop-sg--hlxsites.hlx.page/de/
- After: https://issue-29-sticky-navbar--elco-aerotop-sg--hlxsites.hlx.page/de/

### CODE REVIEW

**Comment by solaris007** [comment]
In the original, the box shadow is not the same when scrolled to the top or in an intermediate position. 

Screenshot 2023-01-17 at 13 56 40

File: blocks/header/header.css [hunk 1]
```diff
@@ -2,6 +2,10 @@
 header {
   box-sizing: border-box;
   box-shadow: 0 10px 50px -2px rgb(0 0 0 / 14%);
+  width: 100%;
+  position: fixed;
+  top: 0;
+  z-index: 1;
```
Comment (solaris007) [inline blocks/header/header.css]: seems to me that z-index is not required

Comment (ekremney) [reply blocks/header/header.css]: it comes from helix bootstrap `header nav` CSS rules: https://github.com/hlxsites/elco-aerotop-sg/pull/30/files#diff-6fa5aa222338bdb49f73313f429cdde763a5bd21358234988a4aa06122738304L18

I think it makes sure that header stays on top of all content. I have no hard feelings about it though, I can remove it

Comment (solaris007) [reply blocks/header/header.css]: ah ok. in that case lets leave it as a precautionary measure.

**Comment by solaris007** [comment]
i am seeing the following when starting to scroll downwards normally:

https://user-images.githubusercontent.com/1872195/213101811-fc5f65f2-a420-4c96-a3eb-9254de7a3e98.mov

**Review by solaris007 — APPROVED** [review]
Perseverance!

**Pull request merged**